Framework/PythonInterface/test/python/mantid/plots/modest_image/test_modest_image.py

```diff
@@ -0,0 +1,292 @@
+from __future__ import print_function, division
+
+import itertools
+import unittest
+
+import matplotlib
+matplotlib.use("agg")
+
+from matplotlib import pyplot as plt
+import matplotlib.image as mi
+
+import numpy as np
+
+from mantid.plots.modest_image import ModestImage
+run_parameterized_tests = True
```

Comment:
Please add a comment here that the `parameterized` module isn't available on all the platforms we support, specific call-out to RHEL7 wouldn't be out of line.